contracts/cw721-ics/src/contract.rs

```diff
@@ -0,0 +1,162 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{Binary, Deps, DepsMut, Empty, Env, MessageInfo, Response, StdResult};
+// TODO: Readd back and setup CW2 correctly
+// use cw2::set_contract_version;
+use cw721::Cw721ReceiveMsg;
+
+use crate::ContractError;
+use cw721_base::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
+use cw721_base::state::TokenInfo;
+use cw721_base::Cw721Contract;
+
+// version info for migration info
+// TODO: Readd these back and setup CW2 correctly
```

Comment:
Relevant example of how to do this: https://github.com/CosmWasm/cw-nfts/pull/72

Reply:
resolved